readme.md

```diff
@@ -1,5 +1,9 @@
 # Microsoft Azure IoT SDK for .NET
 
+### Help us help you with our IoT SDKs!
+We are running a survey through August 2020 to learn more about your IoT projects and support needs. Our team will use this information to help shape the future of our IoT SDKs, and, if you choose to provide your contact information, we'll include you in our circle of advisors for early feedback. Consider spending ~5 minutes completing **[this survey](https://aka.ms/iotsdksurvey)**-- we'd love to hear from you!
```

Comment:
I would just suggest to break the second sentence for easy reading.
Our team will use this information to help shape the future of our **IoT SDKs.  If** you choose to provide your contact information, we'll include you in our circle of advisors for early feedback.

Reply:
The message is already posted in a lot of places. I'll let it stay the same for consistency across all languages.